SA/design_concept/content/design/design_evaluation_iteration_1_results.typ

```diff
@@ -1,16 +1,10 @@
 #import "../../../style.typ": include_section
 
-= Design Evaluation
-This section documents all answers received from the design evaluation questionnaires and the conclusions drawn from them.
-
-#include_section("design_concept/content/design/questionnaire_answers.typ", heading_increase: 2)
-
-== Design Decision <design_decision>
-
-The received questionnaire answers don't determine a clear winner.
+= Conclusions
+Based on the questionnaire answers for the first three designs, which can be found in @design_iteration_one_survey_results , it can be concluded that there is no clear winner.  //TODO: figure out why the refernced section is displayed wrong
```

Comment:
@fl3pp Do you have an idea why the section reference is displayed as "Section 1." instead of "Section 9"?

Reply:
Yes, it's a bug from the part system.

I'll have to look into this, for now we can just merge the PR

Reply:
(I have an idea how to fix it, but it will take more than 5 minutes)